### PR TITLE
common: add tests for index file

### PR DESCRIPTION
**Pull request by stephenplusplus**

### CODE REVIEW

**Comment by stephenplusplus** [comment]
After this, I'm going to do a release unless there's anything else you want to squeeze into common@0.3.0?

**Comment by callmehiphop** [comment]
@stephenplusplus SGTM

File: packages/common/test/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      './grpc-service-object.js': fakeGrpcServiceObject,
+      './service.js': fakeService,
+      './service-object.js': fakeServiceObject,
+      './stream-router.js': fakeStreamRouter,
+      './util.js': fakeUtil
+    });
+  });
+
+  it('should correctly export the common modules', function() {
+    assert.deepEqual(common, {
```
Comment (stephenplusplus) [inline packages/common/test/index.js]: Chose the deepEqual style of test so that this will fail if we add a service and forget to include it in the test.

Comment (callmehiphop) [inline packages/common/test/index.js]: Good call!

**Comment by stephenplusplus** [comment]
> 99.966%

😱

**Comment by stephenplusplus** [comment]
I'll fix that in this PR.

**Comment by callmehiphop** [comment]
Haha, ok. Looks like some custom type checking went untested. My bad :(

**Comment by stephenplusplus** [comment]
Added tests.

**Pull request merged**